docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -69,4 +69,13 @@ $ kubectl get dataimportcron cron-test -o yaml | grep -B 5 storageClassName
         storageClassName: rook-ceph-block
 ```
 
+### Disconnected Environment
+
+For a cluster installed in a disconnected environment, the default golden images can't be polled because they are out of reach, thus the alert will be fired.  
```

Comment:
```suggestion
For a cluster installed in a restricted network, the default golden images cannot be polled because they are out of reach, thus the alert will be fired.  
```

Reply:
Changing.
But aren't "disconnected environment" and "restricted network" synonyms?